packages/uploadthing/src/sdk/index.ts

```diff
@@ -234,17 +234,20 @@ export class UTApi {
     guardServerOnly();
     const { keyType = this.defaultKeyType } = opts ?? {};
 
-    const responseSchema = S.Struct({
+    class DeleteFileResponse extends S.Class<DeleteFileResponse>(
```

Comment:
changed to this syntax just to get named types when hovering:

![CleanShot 2024-05-21 at 09 42 06](https://github.com/pingdotgg/uploadthing/assets/51714798/9fa437b6-a258-472f-9491-033fdc0928ba)